.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: node_js
+node_js:
+  - "0.10"
```

Comment:
I updated the generator to include 0.12 https://github.com/Springworks/generator-springworks/pull/4